lib/node_modules/@stdlib/blas/ext/base/zfill/docs/repl.txt

```diff
@@ -37,41 +37,26 @@
     > var alpha = new {{alias:@stdlib/complex/float64/ctor}}( 5.0, -5.0 );
     > {{alias}}( x.length, alpha, x, 1 );
     > var z = x.get( 0 );
-    > var re = {{alias:@stdlib/complex/float64/real}}( z )
-    5.0
-    > var im = {{alias:@stdlib/complex/float64/imag}}( z )
-    -5.0
+    // returns <Complex128>[ 5.0, -5.0 ]
```

Comment:
```suggestion
    <Complex128>[ 5.0, -5.0 ]
```

Applies here and elsewhere in this file. We don't use `// returns` in REPL help files.